GETTING_STARTED.md

```diff
@@ -1066,10 +1066,11 @@ Example Rake task:
 namespace :factory_bot do
   desc "Verify that all FactoryBot factories are valid"
   task lint: :environment do
-    if Rails.env.test?
-      DatabaseCleaner.clean_with(:deletion)
-      DatabaseCleaner.cleaning do
+    if Rails.env.test?    
```

Comment:
Can you update the text just below this that says "The provided example above
uses the database_cleaner gem; be sure to add the gem to your Gemfile under the appropriate groups."